util/request.py

```diff
@@ -30,15 +30,27 @@ def decorate(func):
         @wraps(func)
         def wrapper(*args, **kwargs):
             cors_origin = app.config.get("CORS_ORIGIN", "*")
+
+            # if there are multiple CORS_ORIGIN set, then use
+            # the Origin header from the request to set the
+            # correct Allow-Origin
+            if isinstance(cors_origin, list):
```

Comment:
Is this a list, or do you need to split on `,` first?

Reply:
I'd say, either make everything a string, or everything a list. The change of from string to list when there's only one policy vs multiple is awkward. 

Otherwise LGTM

Reply:
yeah that's what I was thinking as well. I'll make everything a list